pkg/controller/volume/scheduling/scheduler_binder_test.go

```diff
@@ -809,9 +809,9 @@ func TestFindPodVolumesWithoutProvisioning(t *testing.T) {
 			shouldFail: true,
 		},
 		"prebound-pvc": {
-			podPVCs:    []*v1.PersistentVolumeClaim{preboundPVC},
```

Comment:
@cofyc can you double check if the change to this test case is expected?

Reply:
yes